app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -111,10 +116,10 @@ export default {
         : this.$t('CONVERSATION.FOOTER.MSG_INPUT');
     },
     isMessageLengthReachingThreshold() {
-      return this.message.length > this.maxLength - 40;
+      return this.message.length > this.maxLength - 50;
     },
-    characterCountIndicator() {
-      return `${this.message.length} / ${this.maxLength}`;
+    charactersRemaining() {
```

Comment:
What happens if the message length is greater than the max allowed length. Would it show `-50 characters remaining?`

Reply:
yeah, i can change to `Limit exceeded`. thoughts?